module/VuFind/src/VuFind/Db/Entity/Feedback.php

```diff
@@ -56,81 +56,81 @@ class Feedback implements FeedbackEntityInterface
     #[ORM\Column(name: 'id', type: 'integer', nullable: false, options: ['unsigned' => true])]
     #[ORM\Id]
     #[ORM\GeneratedValue(strategy: 'IDENTITY')]
-    protected $id;
+    protected int $id;
 
     /**
      * Message
      *
      * @var string
      */
     #[ORM\Column(name: 'message', type: 'text', length: 0, nullable: false)]
-    protected $message;
+    protected string $message;
 
     /**
      * Form data
      *
-     * @var mixed
+     * @var ?array
      */
     #[ORM\Column(name: 'form_data', type: 'json', length: 0, nullable: true)]
-    protected $formData;
+    protected ?array $formData = null;
 
     /**
      * Form name
      *
      * @var string
      */
     #[ORM\Column(name: 'form_name', type: 'string', length: 255, nullable: false)]
-    protected $formName;
+    protected string $formName;
 
     /**
      * Creation date
      *
      * @var DateTime
      */
     #[ORM\Column(name: 'created', type: 'datetime', nullable: false, options: ['default' => 'CURRENT_TIMESTAMP'])]
-    protected $created = 'CURRENT_TIMESTAMP';
+    protected DateTime $created;
```

Comment:
Do we need a constructor to properly initialize created and updated, for consistency with the way AuthHash does things?